src/Components/Missions/Missions.js

```diff
@@ -1,9 +1,88 @@
+/* eslint-disable no-tabs */
+/* eslint-disable no-mixed-spaces-and-tabs */
+/* eslint-disable react/button-has-type */
 import React from 'react';
+import { useSelector } from 'react-redux';
+// import Table from 'react-bootstrap/Table';
+// import { Badge, Button } from 'react-bootstrap';
```

Comment:
- [ ] Please remove this and all other commented code. 
- [ ] Also remove the eslint disable line, its best to fix the issue in the code.

Reply:
This one is also properly addressed